system/sbin/kazoo-haproxy

```diff
@@ -9,8 +9,8 @@ if [ -f /etc/sysconfig/haproxy ]; then
 fi
 
 RETVAL=1
-USER=${HAPROXY_USER:-haproxy}
-BIN_FILE=${HAPROXY_BIN:-/usr/sbin/haproxy}
+USER=${HAPROXY_USER:-root}
+BIN_FILE=${HAPROXY_BIN:-/usr/sbin/haproxy-systemd-wrapper}
```

Comment:
@derek2600hz this will break CentOS 6 ;)

When started with systemd thhe HAPROXY_BIN is an environment var so that should overwrite this.  When starting outside of systemd (even on CentOS 7) it should not be using this wrapper (for example, kazoo-haproxy start).